examples/_toc.yml

```diff
@@ -4,6 +4,18 @@
 format: jb-book
 root: Getting_started
 parts:
+  - caption: pytket documentation
+    chapters:
+    - url: https://tket.quantinuum.com/api-docs/
+      title: pytket API docs
+    - url: pytket extensions <https://tket.quantinuum.com/api-docs/extensions.html> 
+      title: pytket extensions
+    - url: https://tket.quantinuum.com/user-manual
+      title: Manual
+    - url: Example notebooks <https://tket.quantinuum.com/examples>
```

Comment:
Why are you writing a name in the link here as well?

Reply:
Sorry copy/paste error. Fixed in https://github.com/CQCL/pytket/pull/293/commits/c5401174c5360f0036db31e53c5496681a1d13d4.